### ISSUE TITLE
Failed to compile error

### ISSUE THREAD

**Issue by klegan** [insidesherpa/JPMC-tech-task-2#236]
After running npm start, I was taken to Google Chrome but then had the following error: 
<img width="852" alt="Screen Shot 2020-06-22 at 2 40 22 PM" src="https://user-images.githubusercontent.com/54511972/85328409-4f7e6900-b496-11ea-8205-8798d98f6671.png"> I was wondering if anyone else had this error or knew how to fix it? Thank you so much for your help!

**Comment by klegan** [comment]
nevermind I figured it out!

**Comment by cmrice1996** [comment]
Hi - I am having the same problem. How did you fix it?

**Comment by klegan** [comment]
@cmrice1996 When I looked at the node_modules folder, I noticed that the @jpmorganchase/perspective folder was not inside (which is why were getting this error). So I re downloaded then node_modules folder from the troubleshooting page, and when I did that, the @jpmorganchase/perspective folder was now in there. Then, I dragged the redownloaded folder back into JPMC-tech-task-2 folder and removed the old one. Then I was able to do npm start in the terminal and it worked. Hope this helps!
